src/jaguar.toit

```diff
@@ -317,6 +332,16 @@ start-image_ -> bool
 
     if on-stopped: on-stopped.call code
 
+    if interval:
+      remaining-us := interval.in-us - (Time.monotonic-us - start-time)
+      scheduled-callbacks.add (Duration --us=remaining-us+1) --callback=::
+        current-run-number := registry_.get-run-counter name
+        if current-run-number == run-number and registry_.id-by-name_.contains name:
```

Comment:
Add a function to the registry_ to support `contains name`. No need to reach into the privates.